fenwick/src/main/java/org/opencadc/fenwick/DeletedStorageLocationEventSync.java

```diff
@@ -0,0 +1,131 @@
+ *  the GNU Affero General Public        la “GNU Affero General Public
+ *  License as published by the          License” telle que publiée
+ *  Free Software Foundation,            par la Free Software Foundation
+ *  either version 3 of the              : soit la version 3 de cette
+ *  License, or (at your option)         licence, soit (à votre gré)
+ *  any later version.                   toute version ultérieure.
+ *
+ *  OpenCADC is distributed in the       OpenCADC est distribué
+ *  hope that it will be useful,         dans l’espoir qu’il vous
+ *  but WITHOUT ANY WARRANTY;            sera utile, mais SANS AUCUNE
+ *  without even the implied             GARANTIE : sans même la garantie
+ *  warranty of MERCHANTABILITY          implicite de COMMERCIALISABILITÉ
+ *  or FITNESS FOR A PARTICULAR          ni d’ADÉQUATION À UN OBJECTIF
+ *  PURPOSE.  See the GNU Affero         PARTICULIER. Consultez la Licence
+ *  General Public License for           Générale Publique GNU Affero
+ *  more details.                        pour plus de détails.
+ *
+ *  You should have received             Vous devriez avoir reçu une
+ *  a copy of the GNU Affero             copie de la Licence Générale
+ *  General Public License along         Publique GNU Affero avec
+ *  with OpenCADC.  If not, see          OpenCADC ; si ce n’est
+ *  <http://www.gnu.org/licenses/>.      pas le cas, consultez :
+ *                                       <http://www.gnu.org/licenses/>.
+ *
+ *  : 5 $
+ *
+ ************************************************************************
+ */
+
+package org.opencadc.fenwick;
+
+import ca.nrc.cadc.date.DateUtil;
+import ca.nrc.cadc.io.ResourceIterator;
+import ca.nrc.cadc.net.ResourceNotFoundException;
+import ca.nrc.cadc.net.TransientException;
+import java.io.IOException;
+import java.net.URI;
+import java.text.DateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+import org.apache.log4j.Logger;
+import org.opencadc.inventory.DeletedStorageLocationEvent;
+import org.opencadc.inventory.InventoryUtil;
+import org.opencadc.tap.TapClient;
+import org.opencadc.tap.TapRowMapper;
+
+public class DeletedStorageLocationEventSync {
+
+    private static final Logger log = Logger.getLogger(DeletedStorageLocationEventSync.class);
+
+    private static final DateFormat dateFormat = DateUtil.getDateFormat(DateUtil.ISO_DATE_FORMAT, DateUtil.UTC);
+
+    private static final String DELETED_STORAGE_LOCATION_QUERY =
+            "SELECT id, lastModified, metaChecksum FROM inventory.DeletedStorageLocationEvent "
+                + "WHERE lastModified is not null AND lastModified >= '%s' order by lastModified";
+
+    private final TapClient<DeletedStorageLocationEvent> tapClient;
+    private final Date startTime;
+
+    public DeletedStorageLocationEventSync(TapClient<DeletedStorageLocationEvent> tapClient, Date startTime) {
+        this.tapClient = tapClient;
+        this.startTime = startTime;
```

Comment:
startTime is optional: not set in constructor

Reply:
Removed from the constructor.